### PR TITLE
Corrected tutorial's Cartesian path example

### PR DESCRIPTION
**Pull request by nbbrooks**
* updated move_group_interface tutorial's Cartesian path waypoints (example 4) so path is triangular
* updated description of the path

### CODE REVIEW

**Comment by davetcoleman** [comment]
Failing because of https://github.com/ros-planning/moveit_tutorials/pull/191

**Comment by mlautman** [comment]
@nbbrooks If you `git commit --amend` then force push, it will trigger travis to re-run.

File: doc/move_group_interface/src/move_group_interface_tutorial.cpp [hunk 1]
```diff
@@ -247,16 +247,16 @@ int main(int argc, char** argv)
...
 
-  target_pose3.position.z += 0.2;
-  waypoints.push_back(target_pose3);  // up
+  target_pose3.position.z -= 0.2;
+  waypoints.push_back(target_pose3);  // down
 
-  target_pose3.position.y -= 0.1;
-  waypoints.push_back(target_pose3);  // left
+  target_pose3.position.y -= 0.2;
+  waypoints.push_back(target_pose3);  // right
```
Comment (mlautman) [inline doc/move_group_interface/src/move_group_interface_tutorial.cpp]: Technically this is subjective but I'll allow it :stuck_out_tongue_closed_eyes:

Comment (nbbrooks) [reply doc/move_group_interface/src/move_group_interface_tutorial.cpp]: If the tutorial didn't mention the box appearing on the arm's right a little bit later in the lesson I would have left this alone :P

**Comment by mlautman** [comment]
congrats @nbbrooks!

**Pull request merged**